packages/sdk-js/src/GrowthBook.ts

```diff
@@ -551,7 +620,9 @@ export class GrowthBook<
           }
         }
       } else {
-        const undo = this._applyDOMChanges(result.value);
+        const undo = this._ctx.applyDomChangesCallback
+          ? this._ctx.applyDomChangesCallback(result.value, this)
```

Comment:
does the callback need `this` arg? No current use case (edge has a custom bypass), but would be for accessing context options like nonce

Reply:
remove